docs/build_docs.py

```diff
@@ -24,6 +24,8 @@
 import pathlib
 import re
 import textwrap
+import typing
+typing.TYPE_CHECKING = True
```

Comment:
is this intentional? if so we should comment it (I'm guessing it's to force type-only imports?)

Reply:
```suggestion
# For showing the conditional imports and types in `content_types.py` 
typing.TYPE_CHECKING = True
```